e2e/helm/values.yaml

```
@@ -68,11 +68,14 @@ minio:
   rootPassword: minioadmin
   memoryLimit: 256Mi
   persistentVolumeSize: 50Gi
-  # tls:   <-- Not implemented yet
-  #   enabled: false
-  #   certSecret: ""
-  #   publicCrt: public.crt
-  #   privateKey: private.key
+  tls:
+    enabled: false
+    # Name of the k8s secret. Minio requires certs to be stored in secrets
+    # https://github.com/minio/minio/tree/master/docs/tls/kubernetes
+    certSecret: null
+    publicCrt: public.crt
+    privateKey: private.key
```

Can we add `location` to these variables? or maybe fileName?
```suggestion
    publicCrtLocation: public.crt
    privateKeyLocation: private.key
```